### DISCUSSION TITLE
How could I subscribe a signal as a client

### DISCUSSION THREAD

**Discussion by ChenhuiZhang** [diwic/dbus-rs#334] [Q&A] [answered]
Does it support to subscribe a signal in DBus service? If yes, could you show some example? Thanks.

**Comment by diwic** [comment, marked as answer]
New example:
https://github.com/diwic/dbus-rs/blob/master/dbus/examples/match_signal.rs

Old example:
https://github.com/diwic/dbus-rs/blob/master/dbus/examples/unity_focused_window.rs
